js/ui/viewSelector.js

```diff
@@ -141,8 +141,8 @@ const ShowOverviewAction = new Lang.Class({
 const ViewsDisplayLayout = new Lang.Class({
     Name: 'ViewsDisplayLayout',
     Extends: Clutter.BinLayout,
-    Signals: { 'allocated-size-changed': { param_types: [GObject.TYPE_INT,
-                                                         GObject.TYPE_INT] } },
+    Signals: { 'grid-available-size-changed': { param_types: [GObject.TYPE_INT,
+                                                              GObject.TYPE_INT] } },
```

Comment:
Looks like this wouldn't be that easy after all because we only have access to the actor from the layout manager, so I'm ok with leaving this like this for now. Maybe we can clean things up a bit during the next rebase now we know more stuff (tm)